xds/internal/xdsclient/client_new.go

```diff
@@ -155,6 +133,28 @@ func NewWithBootstrapContentsForTesting(contents []byte) (XDSClient, func(), err
 	}), nil
 }
 
+func getOrMakeClient(config []byte) (*clientRefCounted, error) {
```

Comment:
Maybe add a comment that this inc's the ref of the client returned but doesn't dec it or provide a function for doing so.

Reply:
Done. Thanks.